packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -113,7 +119,14 @@ export function getBeaconPoolApi({
               metrics?.opPool.attestationPoolInsertOutcome.inc({insertOutcome});
             }
 
-            chain.emitter.emit(routes.events.EventType.attestation, attestation);
+            if (isForkPostElectra(fork)) {
```

Comment:
My general thought about the eventstream is that the `attestation` topic will be deprecated post-electra as it anyways only supports `phase0.Attestation`.

We could consider emitting a `single_attestation` events even pre-electra, this requires to convert the types but it would allow consumers to switch to the new topic earlier and they don't need to keep track of fork epoch to switch.

Reply:
I think we can use the term `SingleAttestation` to denote unaggregated attestations even before electra throughout our codebase

Reply:
Yeah I think it's good semantic wise to do that and likely less confusing.